src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -32,7 +32,7 @@ protected override object OnDotNetInvocationException(Exception exception, strin
                 return base.OnDotNetInvocationException(exception, assemblyName, methodIdentifier);
             }
 
-            var message = $"There was an exception invoking '{methodIdentifier} on assembly {assemblyName}'. For more details turn on " +
+            var message = $"There was an exception '{exception.GetType().Name}' invoking '{methodIdentifier}' on assembly '{assemblyName}'. For more details turn on " +
```

Comment:
Do we normally disclose the exception type name when we're showing non-detailed errors? Just checking that we're doing the same here as for other comparable cases.

Also if we *do* choose to disclose the type name, I suggest phrasing as `There was an exception of type '{name}' when invoking...` (i.e., add `of type` and `when`).

Reply:
Yeah I'd feel better just saying "an exception occurred blah blah..."

Reply:
Fair enough, lets remove that

Reply:
done